### ISSUE TITLE
Window top bar menu options are not accessible via keyboard

### ISSUE THREAD

**Issue by sdellis** [ProjectMirador/mirador#2029]
<img width="375" alt="screen shot 2019-03-01 at 5 11 52 pm" src="https://user-images.githubusercontent.com/609368/53669852-8ea3fd80-3c46-11e9-805d-f9ee7941dc8b.png">

Menu dropdown should also be constrained to its parent window and the layout should be closer to the original mockup as seen here:

<img width="1040" alt="screen shot 2019-03-01 at 5 25 01 pm" src="https://user-images.githubusercontent.com/609368/53669998-0c680900-3c47-11e9-8541-e14730e825b7.png">

**Comment by jvine** [comment]
See also #1936 re placement.

**Comment by camillevilla** [comment]
After discussing this with @ggeisler and @jvine, it may be better to rip out the radio buttons in this menu and replace the options with buttons to make them tab accessible.

**Comment by jvine** [comment]
The keyboard action of this menu should be:
- **tab** to the menu icon, activate it, menu opens
- **arrow key** moves the focus to an item in the menu
- enter (or whatever the "fire" action is in the assistive technology) activates the option
- menu closes automatically.
See the Workspace menu for example.
(Currently in the Window menu: arrow key moves focus, action is activated on focus, window remains open, user has to close the menu as a separate action.)

The mouse/touch behaviour should be:
- click/touch menu icon to open the menu
- click/touch an option in the menu to activate it
- menu closes automatically.
(Currently in the Window menu: click to open, click to activate, window remains open, click outside to close.)

Hidden issue in the current implementation is that once you get past the keyboard block into the menu options, you can use arrow key to navigate, but the radio buttons are firing the action _as soon as they are in focus_.  (To replicate:  Open the menu, click on one of the buttons, use left-right arrows to move between the buttons.) We can't change the user's environment without an explicit action; focus isn't an explicit action. 

![crazybuttons](https://user-images.githubusercontent.com/6945691/54166498-bc492d80-4422-11e9-90e8-2dcbcb72efb7.gif)

Changing the radio buttons to regular buttons *might* help, as long as we don't lose the arrow key menu navigation. Note that there will be other "normal" menu options added below the view/thumb options. 

Material has the concept of a Selection group menu, which might be useful? Grouped options that show selected status. 
https://material.io/develop/web/components/menus/#variants
